### PR TITLE
Feat list the creation time and last update time of each room

### PR DESCRIPTION
**Pull request by aLIEzsss4**

### CODE REVIEW

**Comment by noyyyy** [comment]
I think there is a way to get timestamp outside the chain. I prefer timestamp not recorded on chain

**Comment by aLIEzsss4** [comment]
> I think there is a way to get timestamp outside the chain. I prefer timestamp not recorded on chain

If reset and refresh the browser, can not get block.time

**Comment by noyyyy** [comment]
> > I think there is a way to get timestamp outside the chain. I prefer timestamp not recorded on chain
> 
> If reset and refresh the browser, can not get block.time

We can do this for now, but it's worth a disscussion in mud community.

File: packages/contracts/mud.config.ts [hunk 1]
```diff
@@ -206,6 +206,8 @@ export default mudConfig({
       schema: {
         seatNum: "uint8",
         withPassword: "bool",
+        createdAtBlock: "uint64",
```
Comment (noyyyy) [inline packages/contracts/mud.config.ts:209]: I think `uint32` would be enough.

Comment (aLIEzsss4) [reply packages/contracts/mud.config.ts:209]: With the growth of time and block generation rate, I think uint32 may not be sufficient at some point.

**Review by noyyyy — APPROVED** [review]

**Pull request merged**